lib/jsdom/living/custom-elements/ElementInternals-impl.js

```diff
@@ -15,6 +19,30 @@ class ElementInternalsImpl {
     return shadow;
   }
 
+  get labels() {
+    if (!this._targetElement._ceDefinition.formAssociated) {
+      throw DOMException.create(this._targetElement._globalObject, [
+        "ElementInternals.labels is only supported for form-associated custom elements",
+        "NotSupportedError"
+      ]);
+    }
+    return NodeList.create(this._targetElement._globalObject, [], {
+      element: this._targetElement,
+      query: () => {
+        // TODO: why doesn't .id work? What about attrs that don't reflect idl -> content?
```

Comment:
FYI, it doesn't work because the impl class doesn't have an `id` property, and `_targetElement` is an impl instance. The `id` getter is only part of the generated class, since it's auto-generated to reflect the attribute.